AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Itinerary/Details.cshtml

```diff
@@ -189,6 +189,9 @@
                             }
                             else if (request.Status == RequestStatus.PendingConfirmation)
                             {
+                                <form asp-action="MarkIncomplete" asp-controller="Itinerary" asp-area="Admin" asp-route-itineraryId="@Model.Id" asp-route-requestId="@request.Id" class="right-action-btn">
```

Comment:
@joro550, We don't want to allow the Request status to move to Incomplete (aka, `RequestStatus.Assigned`) when the request's status is `RequestStatus.PendingConfirmation`. It should only be able to move to `Confirmed` or `Unassigned`. 

so, you can remove this line:
```
<form asp-action="MarkIncomplete" asp-controller="Itinerary" asp-area="Admin" asp-route-itineraryId="@Model.Id" asp-route-requestId="@request.Id" class="right-action-btn">
```

This follows the request status "convention" surrounding what requestors who are sent sms confirmation messages can reply back to us with. For more information, you can check out `SmsResponseController`.